utils/notification_service.py

```diff
@@ -1197,7 +1197,7 @@ def pop_default(l: list[Any], i: int, default: Any) -> Any:
             "errors": 0,
             "success": 0,
             "skipped": 0,
-            "time_spent": "",
+            "time_spent": [],
```

Comment:
I change the format, so it is in a better format for processing.
The report duration on Slack would be the total pytest time of all jobs (single/multi gpu runners)